content/en/docs/guides/wait-for-ready.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Wait For Ready
+description: >-
+  Explains how and why to configure the wait for ready feature.
+---
+
+### Overview
+
+This is a feature which can be used on a stub which will cause the rpcs to wait
+(until an optional deadline is exceeded or) for the server to become available
+before sending the request.
+
+This is useful for batch workflows where there is no need to fail fast as it
+allows simplifying the implementation by hiding transient failures.
```

Comment:
Maybe better phrased as below?

"This allows for robust batch workflows as transient server problems won't cause job failures".

Reply:
done